include/aws/http/connection.h

```diff
@@ -286,6 +315,18 @@ enum aws_http_version aws_http_connection_get_version(const struct aws_http_conn
 AWS_HTTP_API
 struct aws_channel *aws_http_connection_get_channel(struct aws_http_connection *connection);
 
+/**
+ * HTTP/2 specific. Change the HTTP/2 conenction settings.
+ * Settings frame will be sent. If set, on_completed callback will always be invoked.
+ */
```

Comment:
If set, on_completed callback will be always be invoked, if no error is reported then the peer has acknowledged the settings.

Reply:
Same as above, `If set, on_completed callback will be always be invoked, if no error is reported then the peer has acknowledged the settings and the change successfully applied.`